src/functions/getTestResultsByTesterStaffId.js

```diff
@@ -15,17 +22,25 @@ const getTestResultsByTesterStaffId = (event) => {
 
   if (!event.queryStringParameters) {
     if (event.queryStringParameters.testerStaffId && event.queryStringParameters.testStationPNumber && event.queryStringParameters.toDateTime && event.queryStringParameters.fromDateTime) {
+      console.log('Bad request in getTestResultsByTesterStaffId - missing required parameters');
```

Comment:
Please move the string in the enum file

Reply:
Is there any point using enums for unique strings? They're only used the once in that specific circumstance